src/chrome/content/scaffold/templates/newWeb.js

```diff
@@ -0,0 +1,75 @@
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 YOUR_NAME
```

Comment:
Probably a good idea to add "TODO" everywhere where adjustments necessary and check it in Travis-CI + it will make it easier in reviewing new translators